runtime/include/chpltypes.h

```diff
@@ -98,11 +98,9 @@ typedef int64_t c_localeid_t;
 // These are special values that mean "no", "any", and "all sublocales",
 // respectively.
 #define c_sublocid_none_val -1
-#define c_sublocid_any_val  -2
```

Comment:
Consider refactoring this so that `all` is `-2`? Given that this is an internal value I would hope nothing is relying on these "magic constants"

Reply:
On a closer look, I think I want to [remove it](https://github.com/Cray/chapel-private/issues/6687) and prefer to leave it untouched to avoid any drama for the time being.

Reply:
Works for me!